unlock-app/src/hooks/useAppStorage.ts

```diff
@@ -40,6 +40,15 @@ export function useAppStorage() {
     return null
   }, [])
 
+  const clearStorageWithoutFavorite = useCallback((): void => {
```

Comment:
No no, this is bad practice, The `useAppStorage` code should not _care_ about what kind of data it stores. 

Instead of that, modify the `clearStorage` function below to accept the list of values it should delete and make sure that when it is called, it only deletes these values.